test_runner/src/main.rs

```diff
@@ -32,7 +32,7 @@ async fn main() {
         } else if test_type == "PAYMENTS_ETH" {
             run_eth_payments_test_scenario()
         } else if test_type == "PAYMENTS_ALTHEA" {
-            run_althea_payments_test_scenario()
+            run_althea_payments_test_scenario().await;
```

Comment:
ok you also need to add this test case to .github/workflows/integration-test.yml so that it runs in CI

Reply:
done